README.md

```diff
@@ -6,10 +6,11 @@ My First Capstone Project
 This project is my first Captone and this is the first project to work on a project for a customer that wants to make a website to announce an event.
 
 ## This is an image for it
-
+![Screenshot from 2022-02-03 00-54-00](https://user-images.githubusercontent.com/92755394/152251439-c8655685-3403-4fa2-91e1-1888fd7573e7.png)
+![Screenshot from 2022-02-03 00-54-14](https://user-images.githubusercontent.com/92755394/152251446-e1484cb9-04dd-41e5-bab7-7032933cba8e.png)
 
 ## Live Demo
-
+[Live Demo Here](https://omar25ahmed.github.io/Omar-Ragheb-First-Capstone/)
 
 
 
```

Comment:
- [ ] The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/), Please, Add this in your readme file